### PR TITLE
CASMINST-4531: Linting, correcting, and reorganizing CANU-related docs

### PR DESCRIPTION
**Pull request by mharding-hpe**
## Summary and Scope

I started out just fixing some small mistakes I saw on one CANU page. Then I noticed some CANU pages existed in two different places, but for no seemingly good reason. This also led me to discover some jarring inconsistencies between the Aruba, Dell, and Mellanox index pages. 

The end result is that I collected all of the CANU-related docs into a new canu subdirectory off the management_network directory, created an index for it, and then updated all of the relevant links. I also did some linting of some (not all) of the pages that were involved. Definitely there is more work to be done here, but this is a first pass.

No meaningful content changes, except for removing some misinformation and adding some additional information in one of the CANU pages, Just information, though, not commands or scripts.

## Issues and Related PRs

There will be backport PRs once this one is reviewed and approved.

## Testing

N/A

## Risks and Mitigations

N/A

## Pull Request Checklist

- [X] License file intact
- [X] Target branch correct

### CODE REVIEW

File: operations/network/management_network/dell/index.md [hunk 1]
```diff
@@ -1,84 +1,81 @@
...
 
 ### Security
-  1. [Access Control Lists (ACLs)](./acl.md)
+
+  * [Access Control Lists (ACLs)](acl.md)
 
 ### Quality of Service
-  1. [Qos)](./qos.md)
+
+  * [Qos)](qos.md)
```
Comment (mharding-hpe) [inline operations/network/management_network/dell/index.md:69]:
```suggestion
  * [Qos](qos.md)
```

File: operations/network/management_network/aruba/index.md [hunk 2]
```diff
@@ -4,202 +4,184 @@
...
-   1. [What is Spine-Leaf Architecture?](./spine_leaf_architecture.md)
-   1. [How Does a Spine-Leaf Architecture Differ From Traditional Network Designs?](./spine_leaf_architecture.md)
-   1. [Why are Spine-Leaf Architectures Becoming More Popular?](./spine_leaf_architecture.md)
-   1. [What is VSX?](./vsx_architecture.md)
-   1. [What are the benefits of VSX?](./vsx_architecture.md)
-   1. [VSX summary](./vsx_architecture.md)
+## Management Network Overview
 
-### Built in High Availability
+   * [Network Types – Naming and segment Function](network_naming_function.md)
```
Comment (nrockershousen) [inline operations/network/management_network/aruba/index.md:52]:
```suggestion
   * [Network Types – Naming and Segment Function](network_naming_function.md)
```

File: operations/network/management_network/aruba/index.md [hunk 3]
```diff
@@ -4,202 +4,184 @@
...
+   * [Network traffic pattern inside of the system](network_traffic_pattern.md)
```
Comment (nrockershousen) [inline operations/network/management_network/aruba/index.md:53]:
```suggestion
   * [Network Traffic Pattern](network_traffic_pattern.md)
```
Syncing procedure titles here with the titles used in each file

File: operations/network/management_network/aruba/index.md [hunk 4]
```diff
@@ -4,202 +4,184 @@
...
+   * [System management network functions in more detail](management_network_function_in_detail.md)
 
-   1. [VSX: MCLAG link HA	](./mclag_link_ha.md)
-   1. [VSX: ISL HA](./mclag_isl_ha.md)
-   1. [VSX: Split](./mclag_split.md)
-   1. [VSX: Member Power Failure](./mclag_power_failure.md)
+## Key Features Used in the Management Network Configuration
 
-### Management Network Overview
+   * [Key Feature list](key_features.md)
```
Comment (nrockershousen) [inline operations/network/management_network/aruba/index.md:58]:
```suggestion
   * [Key Feature List](key_features.md)
```

File: operations/network/management_network/aruba/index.md [hunk 5]
```diff
@@ -4,202 +4,184 @@
...
+   * [Typical configuration of VSX](typical_VSX_configuration.md)
```
Comment (nrockershousen) [inline operations/network/management_network/aruba/index.md:59]:
```suggestion
   * [Typical Configuration of VSX](typical_VSX_configuration.md)
```

File: operations/network/management_network/aruba/index.md [hunk 6]
```diff
@@ -4,202 +4,184 @@
...
+   * [Typical MCLAG port configuration connecting to dual homed devices](typical_mclag_port_configuration.md)
```
Comment (nrockershousen) [inline operations/network/management_network/aruba/index.md:60]:
```suggestion
   * [Typical Configuration of MCLAG Link](typical_mclag_port_configuration.md)
```

File: operations/network/management_network/aruba/index.md [hunk 7]
```diff
@@ -4,202 +4,184 @@
...
+   * [Typical edge port configuration connecting single homed devices](typical_edge_port_configuration.md)
```
Comment (nrockershousen) [inline operations/network/management_network/aruba/index.md:61]:
```suggestion
   * [Typical Edge Port Configuration](typical_edge_port_configuration.md)
```

File: operations/network/management_network/dell/index.md [hunk 8]
```diff
@@ -1,84 +1,81 @@
...
+  * [System images](system_images.md)
 
 ### Layer one features
-  1. [Physical Interfaces](./physical_interfaces.md)
-  1. [Locator LED](./locator_led.md)
+
+  * [Physical Interfaces](physical_interfaces.md)
+  * [Locator LED](locator_led.md)
 
 ### Layer two features
```
Comment (nrockershousen) [inline operations/network/management_network/dell/index.md:45]:
```suggestion
### Layer Two Features
```

File: operations/network/management_network/mellanox/index.md [hunk 9]
```diff
@@ -1,156 +1,146 @@
...
+## Examples of Network topologies
 
-### Quick start guide to Canu
+   * [Very Large](very_large.md)
+   * [Large](large.md)
+   * [Medium](medium.md)
+   * [Small](small.md)
 
-   1. [Quick start guide to Canu](./quick_start_guide_to_canu.md)
+## Network design explained
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:34]:
```suggestion
## Network Design Explained
```

File: operations/network/management_network/mellanox/index.md [hunk 10]
```diff
@@ -1,156 +1,146 @@
...
-   1. [System management network functions in more detail](management_network_function_in_detail.md)
+   * [Key Feature list](key_features.md)
+   * [Typical configuration of mlag switch configuration](typical_mlag_switch_configuration.md)
+   * [Typical MCLAG port configuration connecting to dual homed devices](typical_mlag_port_configuration.md)
 
-### Key features used in the management network configuration
-   1. [Key Feature list](./key_features.md)
-   1. [Typical configuration of mlag switch configuration](./typical_mlag_switch_configuration.md)
-   1. [Typical MCLAG port configuration connecting to dual homed devices](./typical_mlag_port_configuration.md)
+## How to connect management network to your campus network
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:53]:
```suggestion
## How to Connect Management Network to a Campus Network
```

File: operations/network/management_network/mellanox/index.md [hunk 11]
```diff
@@ -1,156 +1,146 @@
...
-   1. [Requirements and optional configuration](./requirements_and_optional_configuration.md)
-   1. [Scenario A: network connection via management network](./scenario-a.md)
-   1. [Scenario B: network connection via high speed network](./scenario-b.md)
-   1. [Example of how to configure Scenario A or B](./management_network_configuration_example.md)
+   * [Requirements and optional configuration](requirements_and_optional_configuration.md)
+   * [Scenario A: network connection via management network](scenario-a.md)
+   * [Scenario B: network connection via high speed network](scenario-b.md)
+   * [Example of how to configure Scenario A or B](management_network_configuration_example.md)
 
 ## Managing switches from CLI
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:60]:
```suggestion
## Managing Switches from the CLI
```

File: operations/network/management_network/mellanox/index.md [hunk 12]
```diff
@@ -1,156 +1,146 @@
...
+  * [PIM-SM Bootstrap Router (BSR) and Rendezvous-Point (RP)](pim.md)
 
 ### Security
-  1. [Access Control Lists (ACLs)](./acl.md)
-  1. [IP filter](./ip_filter.md)
+
+  * [Access Control Lists (ACLs)](acl.md)
+  * [IP filter](ip_filter.md)
 
 ### Performing upgrade on Mellanox
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:114]:
```suggestion
### Performing Upgrade on Mellanox
```

File: operations/network/management_network/mellanox/index.md [hunk 13]
```diff
@@ -1,156 +1,146 @@
...
+  * [Check HSM](check_hsm.md)
+  * [Check Kea DHPC logs](check_kea_dhcp_logs.md)
+  * [TCPDUMP](ncn_tcpdump.md)
+  * [Check BGP and MetalLB](check_bgp_and_metallb.md)
+  * [Getting incorrect IP address. Duplicate IP address check](duplicate_ip.md)
+  * [Large number of DHCP declines during a node boot](dhcp_decline.md)
 
 ### DNS
 
 ### PXE boot
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:137]:
```suggestion
### PXE Boot
```

File: operations/network/management_network/mellanox/index.md [hunk 14]
```diff
@@ -1,156 +1,146 @@
...
-   1. [How does a spine-leaf architecture differ from traditional network designs?](./spine_leaf_architecture2.md)
-   1. [Why are spine-leaf architectures becoming more popular?](./spine_leaf_architecture3.md)
-   1. [What is MLAG?](./mlag_architecture.md)
+## Key features used in the management network configuration
 
-### Management network overview
-   1. [Network Types – Naming and segment Function](./network_naming_function.md)
-   1. [Network traffic pattern inside of the system](./network_traffic_pattern.md)
-   1. [System management network functions in more detail](management_network_function_in_detail.md)
+   * [Key Feature list](key_features.md)
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:49]:
```suggestion
   * [Key Feature List](key_features.md)
```

File: operations/network/management_network/mellanox/index.md [hunk 15]
```diff
@@ -1,156 +1,146 @@
...
+   * [What is MLAG?](mlag_architecture.md)
 
-### Examples of Network topologies:
+## Management network overview
 
-   1. [Very Large](./very_large.md)
-   1. [Large](./large.md)
-   1. [Medium](./medium.md)
-   1. [Small](./small.md)
+   * [Network Types – Naming and segment Function](network_naming_function.md)
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:43]:
```suggestion
   * [Network Types – Naming and Segment Function](network_naming_function.md)
```

File: operations/network/management_network/mellanox/index.md [hunk 16]
```diff
@@ -1,156 +1,146 @@
...
+   * [Network traffic pattern inside of the system](network_traffic_pattern.md)
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:44]:
```suggestion
   * [Network Traffic Pattern](network_traffic_pattern.md)
```

File: operations/network/management_network/mellanox/index.md [hunk 17]
```diff
@@ -1,156 +1,146 @@
...
+   * [System management network functions in more detail](management_network_function_in_detail.md)
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:45]:
```suggestion
   * [System Management Network Functions](management_network_function_in_detail.md)
```

File: operations/network/management_network/mellanox/index.md [hunk 18]
```diff
@@ -1,156 +1,146 @@
...
-   1. [Why are spine-leaf architectures becoming more popular?](./spine_leaf_architecture3.md)
-   1. [What is MLAG?](./mlag_architecture.md)
+## Key features used in the management network configuration
 
-### Management network overview
-   1. [Network Types – Naming and segment Function](./network_naming_function.md)
-   1. [Network traffic pattern inside of the system](./network_traffic_pattern.md)
-   1. [System management network functions in more detail](management_network_function_in_detail.md)
+   * [Key Feature list](key_features.md)
+   * [Typical configuration of mlag switch configuration](typical_mlag_switch_configuration.md)
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:50]:
```suggestion
   * [Typical Configuration of MLAG Between Switches](typical_mlag_switch_configuration.md)
```

File: operations/network/management_network/mellanox/index.md [hunk 19]
```diff
@@ -1,156 +1,146 @@
...
-   1. [Typical configuration of mlag switch configuration](./typical_mlag_switch_configuration.md)
-   1. [Typical MCLAG port configuration connecting to dual homed devices](./typical_mlag_port_configuration.md)
+## How to connect management network to your campus network
 
-### How to connect management network to your campus network
-   1. [Requirements and optional configuration](./requirements_and_optional_configuration.md)
-   1. [Scenario A: network connection via management network](./scenario-a.md)
-   1. [Scenario B: network connection via high speed network](./scenario-b.md)
-   1. [Example of how to configure Scenario A or B](./management_network_configuration_example.md)
+   * [Requirements and optional configuration](requirements_and_optional_configuration.md)
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:55]:
```suggestion
   * [Connect the Management Network to a Campus Network](requirements_and_optional_configuration.md)
```

File: operations/network/management_network/mellanox/index.md [hunk 20]
```diff
@@ -1,156 +1,146 @@
...
+   * [Scenario A: network connection via management network](scenario-a.md)
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:56]:
```suggestion
   * [Scenario A: Network Connection via Management Network](scenario-a.md)
```

File: operations/network/management_network/mellanox/index.md [hunk 21]
```diff
@@ -1,156 +1,146 @@
...
+   * [Scenario B: network connection via high speed network](scenario-b.md)
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:57]:
```suggestion
   * [Scenario B: Network Connection via High-Speed Network](scenario-b.md)
```

File: operations/network/management_network/mellanox/index.md [hunk 22]
```diff
@@ -1,156 +1,146 @@
...
+   * [Example of how to configure Scenario A or B](management_network_configuration_example.md)
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:58]:
```suggestion
   * [Example of How to Configure Scenario A or B](management_network_configuration_example.md)
```

File: operations/network/management_network/mellanox/index.md [hunk 23]
```diff
@@ -1,156 +1,146 @@
...
+  * [Domain Name System (DNS) Client](dns-client.md)
+  * [Exec Banners](exec_banner.md)
+  * [Hostname](hostname.md)
+  * [Domain Name](domain_name.md)
+  * [Secure Shell (SSH)](ssh.md)
+  * [Remote Logging](remote_logging.md)
+  * [Web User Interface (WebUI)](web-ui.md)
+  * [SNMPv2c Community](snmp_community.md)
+  * [SNMPv3 Users](snmpv3_users.md)
+  * [System images](system_images.md)
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:75]:
```suggestion
  * [System Images](system_images.md)
```

File: operations/network/management_network/mellanox/index.md [hunk 24]
```diff
@@ -1,156 +1,146 @@
...
-  1. [MLAG switch configuration](./mlag_switch.md)
-  1. [Multi-Chassis Link Aggregation Group (MCLAG)](./mlag.md)
-  1. [Multiple Spanning Tree Protocol (MSTP)](./mstp.md)
+
+  * [Link Layer Discovery Protocol (LLDP)](lldp.md)
+  * [Virtual Local Access Networks (VLANs)](vlan.md)
+  * [Native VLAN](native_vlan.md)
+  * [VLAN Trunking 802.1Q](vlan_trunking_8021q.md)
+  * [Link Aggregation Group (LAG)](lag.md)
+  * [MLAG switch configuration](mlag_switch.md)
```
Comment (nrockershousen) [inline operations/network/management_network/mellanox/index.md:89]:
```suggestion
  * [MLAG Switch Configuration](mlag_switch.md)
```

**Review by nrockershousen — DISMISSED** [review]

**Review by rustydb — APPROVED** [review]

**Comment by mharding-hpe** [comment]
@gbaker-hpe yesterday you said you wanted some specific people to look over this PR. Who was that? I want to make sure they are on the reviewer list here. I mean, there are no content changes in the PR, so I doubt they'll object to any of it, but I am happy to have folks review it to be sure.

**Review by kosonenj — APPROVED** [review]

**Review by rustydb — APPROVED** [review]

**Comment by mharding-hpe** [comment]
The force pushes this afternoon and evening were just rebases to make sure there was no further linting needed

**Review by rustydb — APPROVED** [review]

**Comment by mharding-hpe** [comment]
Rebased -- no changes. Also, I talked to @trad511 last week. He had planned on reviewing this last Friday, but he did mention that he had looked it over again and didn't think he had any serious concerns.

**Comment by mharding-hpe** [comment]
I see that Sean approved the 1.2 PR, which is identical to this one.

**Review by rustydb — APPROVED** [review]

**Comment by mharding-hpe** [comment]
Working on rebase to fix merge conflict. One sec.

**Comment by mharding-hpe** [comment]
Rebase done.

**Pull request merged**